src/test/ui/autoref-autoderef/deref-into-array.rs

```diff
@@ -0,0 +1,17 @@
+// run-pass
```

Comment:
I don't think these tests need to be run-pass. check-pass should be good enough.